test/e2e/flask/user-operations.spec.ts

```diff
@@ -102,7 +102,16 @@ async function createSwap(driver: Driver) {
     swapFrom: 'TESTETH',
     swapTo: 'USDC',
   });
+
+  await driver.waitForSelector({ text: 'Swap', tag: 'button' });
   await driver.clickElement({ text: 'Swap', tag: 'button' });
+  await driver.assertElementNotPresent(
+    {
+      text: 'View in activity',
+      tag: 'button',
+    },
```

Comment:
it seems that this is still no enough. From the flow we can know that:
- the `View in activity` remains present until the tx is confirmed
- once the tx is confirmed, the `View in activity` disappears and the `Close` button appears

In the failed tests screenshots we can see how we are still in the View activity phase, meaning that the transaction is not yet confirmed. I think we could investigate why the transaction takes so much to be confirmed. Maybe some mock is missing? :thinking: 

![image](https://github.com/user-attachments/assets/ecc6cf47-d357-41a4-9364-3aa85d34f2f3)

Reply:
we can revert this change, as now we've seen that the root cause was that a mock was missing, causing a non-deterministic delay in the API response for the trades